packages/typescript-estree/src/create-program/createWatchProgram.ts

```diff
@@ -104,6 +104,13 @@ function diagnosticReporter(diagnostic: ts.Diagnostic): void {
   );
 }
 
+function createHash(content: string): string {
+  if (ts.sys.createHash) {
```

Comment:
`ts.sys` won't exist in browser environments.

```suggestion
  if (ts.sys && ts.sys.createHash) {
```

Reply:
I don't know about this.